pkgs/native_assets_cli/README.md

```diff
@@ -50,6 +50,23 @@ experiment is only available on the master channel.
 We do breaking changes regularly! So frequently bump `package:native_assets_cli`
 and use dev/master SDK for CI.
 
+**Note:** When configuring `CBuilder` in your `hook/build.dart`, 
```

Comment:
We should not have CBuilder documentation here. The native assets cli can be used with other builders. Add the documentation to the CBuilder instead.